adoc/admin-updates.adoc

```diff
@@ -231,6 +231,10 @@ It is not required to reload or restart `skuba-update.timer`.
 The `--annotate-only` flag makes the `skuba-update` utility only check if updates are available and annotate the node accordingly.
 When this flag is activated no updates are installed at all.
 
+When OS updates are disabled, then you will have to manage OS updates manually. In order to do so, you will have to call `skuba-update` manually on each node.
+
+After that, rebooting the node will depend on whether you have also disabled reboots or not. If you have disabled reboots for this node, then you will have to follow the instructions as given in <<Completely Disabling Reboots>>. Otherwise, you will have to wait until `kured` performs the reboot of the node
+
```

Comment:
Do we have some details on how to adjust kured configuration? I.e. check every 10 Minutes instead of every 60 Minutes? In case no - we should add that - in case yes - we should link this, here.

Reply:
@mssola Can you specify more details for this?